src/ripple/app/misc/NetworkOPs.cpp

```diff
@@ -905,14 +905,17 @@ void NetworkOPsImp::submitTransaction (
         return;
     }
 
+    std::string reason;
```

Comment:
I would be inclined to move the declaration of `reason` down a couple of lines to the interior of the `if`.  I don't think `reason` is needed outside of the `if`'s scope.